eclair-core/src/main/scala/fr/acinq/eclair/channel/Helpers.scala

```diff
@@ -619,11 +624,10 @@ object Helpers {
       * @param tx                       the remote commitment transaction that has just been published
       * @return a list of transactions (one per HTLC that we can claim)
       */
-    def claimRemoteCommitMainOutput(keyManager: KeyManager, commitments: Commitments, remotePerCommitmentPoint: PublicKey, tx: Transaction)(implicit log: LoggingAdapter): RemoteCommitPublished = {
+    def claimRemoteCommitMainOutput(feeTargets: FeeTargets, keyManager: KeyManager, commitments: Commitments, remotePerCommitmentPoint: PublicKey, tx: Transaction, feeEstimator: FeeEstimator)(implicit log: LoggingAdapter): RemoteCommitPublished = {
```

Comment:
`feeTargets` should be after `feeEstimator`

Reply:
Done in f0649dd